security/imx6/gen_close.sh

```diff
@@ -57,11 +68,12 @@ fi
 (cat << EOF
 uuu_version 1.2.39
 
-SDP: boot -f SPL-mfgtool.signed -dcdaddr 0x0907000 -cleardcd
+SDP: boot -f SPL-mfgtool.signed
+$TORADEX
 
 SDPV: delay 1000
-SDPV: write -f u-boot-mfgtool.itb
-SDPV: jump
+SDPV: write -f u-boot-mfgtool.itb -addr 0x12000000
+SDPV: jump -addr 0x12000000
```

Comment:
I believe we were going to add this address in lmp.cfg CONFIG_SDP_LOADADDR and remove from the uuu scripts, what is the best approach?

Reply:
after discussion with Mike, we put it here initially and once all SPLs have been updated then we can later remove it. But it does  not hurt to have it in 2 places as long as it is the same